RUFAS/graph_generator.py

```diff
@@ -214,6 +218,42 @@ def generate_graph(
 
         return all_logs
 
+    def _add_var_units(
+        self, filtered_pool: dict[str, List[Any]]
+    ) -> Tuple[dict[str, List[Any]], list[dict[str, str | dict[str, str]]]]:
+        """Adds variable units to variable name for graphing.
+
+        Parameters
+        ----------
+        filtered_pool : dict[str, List[Any]]
+            The data to be graphed.
+
+        Returns
+        -------
+        Tuple[dict[str, List[Any]], list[dict[str, str | dict[str, str]]]]
+            The updated data with units added and logs if info_maps aren't found to get units.
+        """
+        updated_data = {}
+        info_map = {
+            "class": self.__class__.__name__,
+            "function": self._add_var_units.__name__,
+        }
+        for var_name, details in filtered_pool.items():
+            if "info_maps" in details and details["info_maps"]:
+                unit = details["info_maps"][0]["units"]
+                new_var_name = f"{var_name} ('{unit}')"
```

Comment:
```suggestion
                new_var_name = f"{var_name} ({unit})"
```
I think the quotes aren't really necessary here.